client/util/util_test.go

```diff
@@ -15,6 +15,7 @@ package util
 
 import (
 	"testing"
+	"os"
```

Comment:
FYI, golang convention requires imports to be in alphabetic order (if grouped together) - you can use the command "gofmt -w util_test.go" to format+update the file in-place with the right formatting.

Reply:
Ah okay, that makes sense. I'll be sure to run that command in the future!